src/main/java/seedu/address/logic/commands/DietCommand.java

```diff
@@ -59,7 +58,6 @@ public CommandResult execute(Model model) throws CommandException {
                 petToEdit.getAttendanceHashMap());
 
         model.setPet(petToEdit, editedPet);
-        model.updateFilteredPetList(PREDICATE_SHOW_ALL_PETS);
```

Comment:
Is there a reason why you deleted this line? Since not all of the current Commands have this behaviour!

Reply:
yep just realized this! thanks for pointing it out!